sdk/servicebus/Microsoft.Azure.ServiceBus/src/QueueClient.cs

```diff
@@ -151,6 +151,54 @@ public QueueClient(ServiceBusConnection serviceBusConnection, string entityPath,
             MessagingEventSource.Log.QueueClientCreateStop(serviceBusConnection.Endpoint.Authority, entityPath, this.ClientId);
         }
 
+        /// <summary>
+        /// Creates a new instance of the Queue client using Azure Active Directory authentication.
+        /// </summary>
+        /// <param name="endpoint">Fully qualified domain name for Service Bus. Most likely, {yournamespace}.servicebus.windows.net</param>
+        /// <param name="entityPath">Queue path.</param>
+        /// <param name="authCallback">User provided delegate that will provide the access token.</param>
+        /// <param name="authority">Address of the authority to issue token.</param>
+        /// <param name="transportType">Transport type.</param>
+        /// <param name="receiveMode">Mode of receive of messages. Defaults to <see cref="ReceiveMode"/>.PeekLock.</param>
+        /// <param name="retryPolicy">Retry policy for queue operations. Defaults to <see cref="RetryPolicy.Default"/></param>
+        /// <remarks>Creates a new connection to the queue, which is opened during the first send/receive operation.</remarks>
+        public static QueueClient CreateWithAzureActiveDirectory(
```

Comment:
>static [](start = 15, length = 6)

Even if this API is needed, this should not be static. This SDK has slightly different API syntax compared to the older one. Here you should be able to do a `new QueueClient()` , and not `QueueClient.CreateWith*`

Reply:
If we do constructor, the signature will collide with the one below. Eventhubs is also using Create() methods

---
In reply to: [286696743](https://github.com/Azure/azure-sdk-for-net/pull/6393#discussion_r286696743) [](ancestors = 286696743)